features/rt/assets/rt-kernel-patch.sh

```diff
@@ -20,15 +29,26 @@ if [[ $replacedPackages =~ "microcode_ctl" ]]
 then
     echo "microcode_ctl patch already installed"
 else
+    echo "Installing microcode_ctl patch"
     rpm-ostree override replace ${MICROCODE_URL}
 fi
 
 # Swap to RT kernel
 kernel=$(uname -a)
 if [[ $kernel =~ "PREEMPT RT" ]]
 then
-    # TODO: check for RT kernel updates
-    echo "RT kernel already installed"
+    echo "RT kernel already installed, checking for updates"
+    rpm-ostree upgrade --unchanged-exit-77
```

Comment:
i think this unchanged-exit-77 makes the script exits with a non zero status after initial kernel patching, and as suycj systemd unit shows as failed

Reply:
the initial kernel patch should not go through this code branch, see the check if kernel contains "PREEMPT RT" :thinking:

Reply:
ok, the rt repo URL (which I got from docs) does not seem to work, the $basearch var isn't resolved. Didn't notice this earlier because my server isn't on VPN, so I used a repo copy with another URL. Sorry. Fixed.

Reply:
if you still run into a problem, please provide logs of the systemd unit (`journalctl -u rt-kernel`), thanks

Reply:
@karmab anything still missing from your POV?